Benzy-Clubbers/benzy-clubbers-ember
# Added Car details demo page with Reviews. Fetching data to list cars, display car details and reviews

What I Did:
- Crated demo to list all cars (sorted by id) on the home page. These are links
- Created demo of details page that displays info about the car and Reviews (if any)

How to Test:
- Run the backend with the most recent updates
- Run ember `ember serve`
- On the home page (index page) you should see 29 links to different cars
- Click on the first one as it's one of the only one's having demo reviews
- You must see car details and a list of reviews

All changes have been added for demo purposes, it wasn't intended to have any of the UI.
